src/lib.rs

```diff
@@ -4,6 +4,7 @@
 
 pub use miette_derive::*;
 
+pub use chain::*;
```

Comment:
Is this necessary?

Reply:
Yes, I think exporting named opaque type in [chain method](https://docs.rs/miette/latest/miette/struct.Error.html#method.chain) is better than exporting private type.

Alternative is to change `chain` to return `impl Something`.